### ISSUE TITLE
"Unknown provider" running as built binary

### ISSUE THREAD

**Issue by spadgett** [openshift/origin-web-console#2682]
I believe this is from the osc-webhook-triggers component dependency injection. It should be look like

https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/overview/listRow.js#L5

```
vendor.js:5659 Error: [$injector:unpr] Unknown provider: eProvider <- e
http://errors.angularjs.org/1.5.11/$injector/unpr?p0=eProvider%20%3C-%20e
    at vendor.js:3372
    at vendor.js:4147
    at Object.i [as get] (vendor.js:4085)
    at vendor.js:4149
    at i (vendor.js:4085)
    at r (vendor.js:4096)
    at Object.invoke (vendor.js:4106)
    at c.instance (vendor.js:5086)
    at f (vendor.js:4624)
    at vendor.js:4805
```

/kind bug
/priority p1
/assign @jhadvig 

cc @jwforres

**Comment by jwforres** [comment]
@spadgett yeah @jhadvig already has a fix as part of https://github.com/openshift/origin-web-console/pull/2679/files

**Comment by spadgett** [comment]
Fixed by #2679 
/close
